ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -204,63 +174,55 @@ export const AdminUserTable = withUserProfile()(
       }
     }
 
-    convertProfilesToFields(users: AdminTableUser[]) {
-      return users.map((user) => ({
-        bypass: (
-          <AdminUserBypass
-            user={{ ...user }}
-            onBypassModuleUpdate={() => this.loadProfiles()}
-          />
-        ),
-        complianceTraining: this.accessModuleCellContents(
-          user,
-          'complianceTraining'
-        ),
-        ctComplianceTraining: this.accessModuleCellContents(
-          user,
-          'ctComplianceTraining'
-        ),
-        contactEmail: (
-          <a href={`mailto:${user.contactEmail}`}>{user.contactEmail}</a>
-        ),
-        dataAccess: this.dataAccessContents(user),
-        dataUseAgreement: this.accessModuleCellContents(
-          user,
-          'dataUseAgreement'
-        ),
-        enabled: user.disabled ? <DisabledIcon /> : <EnabledIcon />,
-        eraCommons: this.accessModuleCellContents(user, 'eraCommons'),
-        rasLinkLoginGov: this.accessModuleCellContents(user, 'rasLinkLoginGov'),
-        firstSignInTime: this.formattedTimestampOrEmptyString(
-          user.firstSignInTime
-        ),
-        firstSignInTimestamp: user.firstSignInTime,
-        institutionName: this.displayInstitutionName(user),
-        name: (
-          <AdminUserLink username={user.username} target='_blank'>
-            {user.familyName + ', ' + user.givenName}
-          </AdminUserLink>
-        ),
-        // used for filter and sorting
-        nameText: user.familyName + ' ' + user.givenName,
-        twoFactorAuth: this.accessModuleCellContents(user, 'twoFactorAuth'),
-        username: (
-          <AdminUserLink username={user.username} target='_blank'>
-            {usernameWithoutDomain(user.username)}
-          </AdminUserLink>
-        ),
-      }));
+    convertProfilesToFields(users: AdminTableUser[]): DataTableFields[] {
+      return users
+        .map((user) => ({
+          // 'text' fields are used for sorting
+
+          name: (
```

Comment:
these fields are now in the same order as the table fields instead of alphabetized.

`text` fields (like nameText) are strings used to help in sorting the component fields (like name)